### DISCUSSION TITLE
dagre布局的节点怎么居中对齐

### DISCUSSION THREAD

**Discussion by Y-HR** [antvis/X6#3735] [Q&A] [answered]
这里说的居中对齐指的是在同一水平线的节点的中心点是对齐的，如果节点的宽高不一致再加上使用manhattan路由，相邻节点之间会出现不必要的折线。![image](https://github.com/antvis/X6/assets/37690977/4af00612-eead-4f40-b641-0e1cceb43b38)

**Comment by NewByVector** [comment, marked as answer]
![image](https://github.com/antvis/X6/assets/20186737/0c2dc16a-74cd-40f0-8852-8a9bf19bfa2a)
